common/gratia/common/sandbox_mgmt.py

```diff
@@ -569,21 +569,29 @@ def OpenNewRecordFile(dirIndex):
             if not os.path.exists(working_dir):
                 try:
                     Mkdir(working_dir)
-                except:
-                    continue
+                except Exception as exc:
+                    msg = 'Warning: Exception caught while creating directory: ' + working_dir
```

Comment:
```suggestion
                    msg = 'Fatal: Exception caught while creating directory: ' + working_dir
```
if you're bailing, it's not a warning.

Reply:
Good point